x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -115,8 +115,41 @@ cd x-pack/plugins/security_solution
 CYPRESS_BASE_URL=http(s)://<username>:<password>@<kbnUrl> CYPRESS_ELASTICSEARCH_URL=http(s)://<username>:<password>@<elsUrl> CYPRESS_ELASTICSEARCH_USERNAME=<username> CYPRESS_ELASTICSEARCH_PASSWORD=password yarn cypress:run:firefox
 ```
 
+#### CCS Custom Target + Headless
+
+This test execution requires two clusters configured for CCS. See [Search across clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html) for instructions on how to prepare such setup.
+
+The instructions below assume:
+* Search cluster is on server1
+* Remote cluster is on server2
+* Remote cluster is accessible from the search cluster with name 'data'
+* Security and TLS are enabled
+
+```shell
+# bootstrap kibana from the project root
+yarn kbn bootstrap
+
+# build the plugins/assets that cypress will execute against
+node scripts/build_kibana_platform_plugins
```

Comment:
For a custom target, this step it is not needed :)

Reply:
Not so sure, I ended up in such situation that I had to rebuild the plugins otherwise the test failed for the wrong reason.

Reply:
Resolved